configure.ac

```diff
@@ -83,14 +83,25 @@ AC_ARG_WITH([lustre],
         [AS_HELP_STRING([--with-lustre],
                 [support configurable Lustre striping values @<:@default=check@:>@])],
         [], [with_lustre=check])
-AS_IF([test "x$with_lustre" != xno], [
```

Comment:
What was the motivation behind this change?  It looks like `--with-gpfs` right above is still using the `!= xno`.

Reply:
Everything that isn't explicitly set to "yes" or (= xyes) is **!= xno**. Even if garbage data is somehow pulled in. I was experiencing issues earlier on in my testing because of this, when I was **not** building --with-lustre and it was still falling into that same code path. To maintain form, the others can be modified as part of this or another PR. Or if you prefer, I can revert while to discuss our options.

Reply:
@glennklockwood: any additional thoughts on this?